index.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
```

Comment:
**praise**: The title is descriptive and reflects the technologies used, which is helpful for understanding the project's purpose.